ims/cli/cli.py

```diff
@@ -2,33 +2,36 @@
 
 import json
 import sys
-
+import os
 import click
 import requests
 from prettytable import PrettyTable
 
-import ims.common.config as config
 import ims.common.constants as constants
+import ims.common.config as config
+
 
 config.load()
 
 from ims.einstein.operations import BMI
-from ims.database import *
+# from ims.database.database import Database
+from ims.exception.exception import BMIException
 
 _cfg = config.get()
 
-_url = "http://{0}:{1}/".format(_cfg.bind_ip, _cfg.bind_port)
+_url = "http://{0}:{1}/".format(_cfg.http[constants.BIND_IP_KEY],
+                                int(_cfg.http[constants.BIND_PORT_KEY]))
 
-if constants.HAAS_USERNAME_VARIABLE in os.environ:
-    _username = os.environ[constants.HAAS_USERNAME_VARIABLE]
+if constants.HIL_USERNAME_VARIABLE in os.environ:
```

Comment:
I meant environment variable. I'll add an ENV in the name to make it more clear.